cocos/renderer/gfx-gles2/GLES2Fence.cc

```diff
@@ -0,0 +1,49 @@
+#include "GLES2Std.h"
+#include "GLES2Fence.h"
+#include "GLES2Device.h"
+#include "GLES2GPUObjects.h"
+
+NS_CC_BEGIN
+
+GLES2Fence::GLES2Fence(GFXDevice* device)
+    : GFXFence(device)
+{
+}
+
+GLES2Fence::~GLES2Fence()
+{
+}
+
+bool GLES2Fence::initialize(const GFXFenceInfo &info)
+{
+    _gpuFence = CC_NEW(GLES2GPUFence);
```

Comment:
It is better to check result.

Reply:
Other use cases didn't do that, so I kept the same convention here.

Reply:
Yep, we should modify them if found. Check memory allocation result is a good practice, it makes codes more robust.